src/features/authentication/staticConfigs.ts

```diff
@@ -0,0 +1,65 @@
+import * as msal from "@azure/msal-browser";
```

Comment:
1. split Auth0 and MSAL in separate files.
2. this are fine _only_ if are _defaults_ which are partially overridden in Main / Config during startup, otherwise remove them from the 'authentication' library code.